src/config/fonts/styles.css

```diff
@@ -1,96 +1,36 @@
 @font-face {
-  font-family: 'Arizona Serif';
-  src: url('./ABCArizonaSerif-Light.ttf') format('truetype');
+  font-family: 'ArizonaSerif';
+  src: url('./ArizonaSerif/ArizonaSerif-Light.otf') format('truetype');
+  font-weight: 200;
   font-style: normal;
 }
-
-@font-face {
-  font-family: 'Arizona Serif';
-  src: url('./ArizonaSerif-Light.woff') format('woff');
-  font-style: normal;
-}
-
-@font-face {
-  font-family: 'Arizona Serif';
-  src: url('./ArizonaSerif-Light.woff2') format('woff2');
-  font-style: normal;
-}
-
-@font-face {
-  font-family: 'Arizona Text';
-  src: url('./ArizonaText-Light.ttf') format('truetype');
-  font-style: normal;
-}
-
-@font-face {
-  font-family: 'Arizona Text';
-  src: url('./ArizonaText-Light.woff') format('woff');
-  font-style: normal;
-}
-
-@font-face {
-  font-family: 'Arizona Text';
-  src: url('./ArizonaText-Light.woff2') format('woff2');
-  font-style: normal;
-}
-
-@font-face {
-  font-family: 'Arizona Text';
-  src: url('./ArizonaText-LightItalic.ttf') format('truetype');
-  font-style: italic;
-}
-
-@font-face {
-  font-family: 'Arizona Text';
-  src: url('./ArizonaText-LightItalic.woff') format('woff');
-  font-style: italic;
-}
-
-@font-face {
-  font-family: 'Arizona Text';
-  src: url('./ArizonaText-LightItalic.woff2') format('woff2');
-  font-style: italic;
+/* @font-face {
```

Comment:
I am wondering why this is commented out?

Reply:
Done

Reply:
We will need to check these imports when exporting to NPM.